### PR TITLE
Basic info on tigris.princeton.edu.

### PR DESCRIPTION
**Pull request by kevinreiss**

### CODE REVIEW

File: services/hosted_services.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Groups defined
+* TIGRIS Administrator 1 User (Anne Marie)
+* TIGRIS Legal User
+* TIGRIS Read Write
+* TIGRIS Storage User
+* TIGRIS User  
+
+### Renewing the SSL Cert
+Tigris.princeton.edu has an A DNS record pointing to: princetonuniversity.cloudapp.net. When the certificate is expiring a new one needs to be emailed to support for tigris at: support@gimmal.com. Must be submitted as a .pfx file.
```
Comment (acozine) [inline services/hosted_services.md:19]: Not a blocker, but I'm curious: how does gimmal.com relate to aisww.com? Are we leasing software from aisww and buying support from gimmal? or is one a subsidiary of the other or something?

File: services/hosted_services.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Hosted Systems in Use at PUL
+
+## tigris.princeton.edu
+
+### Purpose
+Records management software used by both University staff and Library staff in the Library's Records Management. Anne-Marie Phillips is the primary contact for the software. The software product name for this is "Infolinx". Further information available at https://aisww.com/infolinx/. 
+
+### Access
+Visit https://tigris.princeton.edu. Tigris is integrated with our campus IDP. Access to it is governed by membership in certain active directory works established specifically for tigris. These need to be managed via https://tools.princeton.edu.
```
Comment (acozine) [inline services/hosted_services.md:9]:
```suggestion
Visit https://tigris.princeton.edu. Tigris is integrated with our campus IDP. Access to it is governed by membership in certain Active Directory groups established specifically for tigris. These need to be managed via https://tools.princeton.edu. 
```

**Review by acozine — APPROVED** [review]
This is awesome! I put in one small edit and a question, but I'd be happy to merge this any time.

File: services/hosted_services.md (hunk 1 above)
Comment (kevinreiss) [reply services/hosted_services.md:19]: Aisww is the vendors website that the infolinx brand is attached to. https://gimmal.com/ appears to be another type of records management software. Maybe gimmal bought them out and provides support now.

**Pull request merged**